react-ai-chatbot/src/components/Controls/Controls.jsx

```
@@ -30,12 +37,14 @@ export function Controls({onSend}) {
     return (
         <div className={styles.Controls}>
             <div className={styles.TextAreaContainer}>
-                <TextareaAutosize minRows={1} maxRows={10} placeholder="Message AI Chatbot"
+                <TextareaAutosize ref={textareaRef} disabled={disabled} minRows={1} maxRows={10}
+                                  placeholder="Message AI Chatbot"
                                   aria-label="Message AI Chatbot" cacheMeasurements className={styles.TextArea}
                                   value={content}
                                   onChange={handleContentChange} onKeyDown={handleEnterPress}></TextareaAutosize>
             </div>
-            <button className={styles.Button} onClick={handleContentSend}><SendIcon></SendIcon></button>
+            <button disabled={disabled} className={styles.Button} onClick={handleContentSend}><SendIcon></SendIcon>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)